test.md

```diff
@@ -180,33 +171,153 @@ only need to do it on the first one. If you have a `<th>`, set it there.
   </tr>
   <tr>
   <td>
-  <a class="button outline-btn" href="/">Go to the docs!</a><br><br>
-  <a href="/"><font color="#BDBDBD" size="-1">It is dark here. You are likely to be eaten by a grue.</font></a>
+  <p><a class="button outline-btn" href="/">Go to the docs!</a><</p>
+  <p><a href="/"><font color="#BDBDBD" size="-1">It is dark here. You are likely to be eaten by a grue.</font></a></p>
   </td>
   <td>
-  <a class="button outline-btn" href="/">Go to the docs!</a><br><br>
-  <a href="/"><font color="#BDBDBD" size="-1">It is dark here. You are likely to be eaten by a grue.</font></a>
+  <p><a class="button outline-btn" href="/">Go to the docs!</a></p>
+  <p><a href="/"><font color="#BDBDBD" size="-1">It is dark here. You are likely to be eaten by a grue.</font></a></p>
   </td>
   </tr>
 </table>
 
-##### Heading 5
+## Mixing Markdown and HTML
 
-This heading is not included in the right-nav. To include it set `toc_max: 5` in
-the page's front-matter.
+You can use <b>span-level</b> HTML tags within Markdown.
 
-Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+You can use a `<br />` tag to impose an extra newline like here:<br />
 
-###### Heading 6?!
+You can use entities like `&nbsp;` to keep a&nbsp;bunch&nbsp;of&nbsp;words&nbsp;together&nbsp;.
 
-This heading is not included in the right-nav. To include it set `toc_max: 6` in
-the page's front-matter.
+<center>
+You can use block-level HTML tags too. This paragraph is centered.
+</center>
+
+Keep reading for more examples, such as creating tabbed content within the
+page or displaying content as "cards".
+
+## Jekyll / Liquid tricks
+
+This paragraph is centered and colored green by setting CSS directly on the element.
+{: style="text-align:center; color: green" }
```

Comment:
I don't think we want to encourage contributors to add CSS styles. I agree with you in principal, but ...